cmd/plugins/drone/main.go

```diff
@@ -84,67 +89,127 @@ func action(c *cli.Context) error {
 		return fmt.Errorf("failed to parse SignalCD config: %w", err)
 	}
 
-	apiURLFlag := c.String("api.url")
+	apiURLFlag := c.String(flagAPIURL)
 	if apiURLFlag == "" {
-		return xerrors.New("no API URL provided")
+		return fmt.Errorf("no API URL provided")
```

Comment:
We don't have formatting better to use `errors.New`
```suggestion
		return errors.New("no API URL provided")
```